src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -494,8 +494,11 @@
     <value>Restoring NuGet packages...
 To prevent NuGet from downloading packages during build, open the Visual Studio Options dialog, click on the Package Manager node and uncheck '{0}'.</value>
   </data>
-  <data name="RestoreCommandNoPackagesConfigOrProjectJson" xml:space="preserve">
-   <value>Unable to restore. The project does not contain a project.json or packages.config file.</value>
+  <data name="ProjectRestoreCommandNoPackagesConfigOrProjectJson" xml:space="preserve">
+   <value>Nothing to do. The project does not contain a project.json or packages.config file.</value>
```

Comment:
to be very accurate it should include the packages.config with project name and project.json with project name.